src/components/cc-ssh-key-list/cc-ssh-key-list.stories.js

```diff
@@ -33,6 +33,19 @@ const DUMMY_KEY_3 = {
   fingerprint: '00:03:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:7f:62',
 };
 
+const baseItem = {
+  keyData: {
+    state: 'loaded',
+    isGithubLinked: true,
+    personalKeys: [DUMMY_KEY_1],
+    githubKeys: [DUMMY_KEY_2],
+  },
+};
+
+/**
+ * @typedef {import('./cc-ssh-key-list.js').CcSshKeyList} CcSshKeyList
+ */
```

Comment:
fix: need to move these after `export default` because it breaks storybook docs  (I don't have a better solution since it's related to Storybook's code)